packages/components/src/components/hds/stepper/task/indicator.ts

```diff
@@ -6,24 +6,34 @@
 import Component from '@glimmer/component';
 import { assert } from '@ember/debug';
 
-export const DEFAULT_STATUS = 'incomplete';
-export const STATUSES = ['incomplete', 'progress', 'processing', 'complete'];
-export const MAPPING_STATUS_TO_ICONS = {
-  incomplete: 'circle',
-  progress: 'circle-half',
-  processing: 'loading',
-  complete: 'check-circle',
-};
-
-export default class HdsStepperIndicatorTaskIndexComponent extends Component {
+import {
+  HdsStepperStatusesValues,
+  HdsStepperStatusToIconsValues,
+} from '../types.ts';
+import type { HdsStepperStatuses } from '../types.ts';
+
+export const DEFAULT_STATUS = HdsStepperStatusesValues.Incomplete;
+export const STATUSES: string[] = Object.values(HdsStepperStatusesValues);
+
+export const MAPPING_STATUS_TO_ICONS = HdsStepperStatusToIconsValues;
+
+interface HdsStepperIndicatorTaskSignature {
+  Args: {
+    isInteractive?: boolean;
+    status: HdsStepperStatuses;
```

Comment:
See: https://helios.hashicorp.design/components/stepper?tab=code#steppertaskindicator
```suggestion
    status: HdsStepperStatuses;
    isInteractive?: boolean;
```